3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -229,6 +229,33 @@ I dati necessari alla creazione sono:
 Successivamente all'inserimento dei dati rappresentativi della nuova zona personalizzata, è possibile premere sul pulsante "Crea zona" per generare l'elemento 3D corrispondente nell'ambiente di lavoro.
 Esso verrà posizionato automaticamente alle coordinate (0,0) del piano e sarà quindi possibile riposizionarlo.
 
+== Impostazioni
+
+#figure(
+    grid(
+        columns: 2,
+        rows:    (auto, auto),
+        [ #image("./imgs/impostazioni.png", width: 50%)],
+        [ #image("./imgs/ridimensionamento.png", width: 90%)],
+    ),caption: [Impostazioni ambiente 3D],
+) <impostazioni>
+
+Premendo il pulsante *Setting* in basso a sinistra dello schermo si aprirà il pannello mostrato in @impostazioni (sinistra).
+
+In alto troviamo *Informazioni*, contenente varie informazioni riguardo il prodotto.
+
+Nella sezione subito sottostante troviamo *Planimetria* con riportati i valori dimensionali del piano, qui è possibile, cliccando sui valori mostrati, ridimensionarne le dimensioni.
+
+L'inserimento di valori che andrebbero a rimpicciolire il piano vengono considerati errati e la modifica delle dimensioni resa impossibile.
+
+Una volta modificati i valori e premuto il pulsante *Salva* viene mostrato il pannello in @impostazioni (destra) che mostra in bianco il piano attuale e tratteggiata l'estensione che si vuole apportare. Premere quindi *Conferma* per applicare le modifiche.
+
+Da notare che nel caso di modifica di un Piano Personalizzato il ridimensionamento non andrà ad influire sul SVG che manterrà le dimensioni definite in fase di inizializzazione, ma aumenterà la superficie bianca su cui è disegnato.
+
+Nell'ultima sezione in basso, *Demo*, troviamo due pulsanti:
+- *Risincronizza* che permette riportare il piano al suo stato iniziale, eliminando quindi tutte le zone create e gli spostamenti effettuati;
```

Comment:
- Toglierei "che" e metterei ":";
- Cambierei in "... stato iniziale, annullando tutte le modifiche e gli spostamenti effettuati;".